src/components/IncidentTable/IncidentTableComponent.jsx

```diff
@@ -369,32 +349,15 @@ const IncidentTableComponent = () => {
     ({
       data, index, style,
     }) => {
-      const {
-        ref, inView,
-      } = useInView({
-        threshold: 0.1,
-      });
-
       const row = data[index];
-      useEffect(() => {
-        if (inView) {
-          if (!row.original.alerts) {
-            getIncidentAlerts(row.original.id);
-          }
-          if (!row.original.notes) {
-            getIncidentNotes(row.original.id);
-          }
-        }
-      }, [inView]);
```

Comment:
@martindstone - need to review this as the change we did to fix the alert count in #321 was implemented here in this `useEffect`, but it's removed in this PR. Is it handled elsewhere instead?

```javascript
 useEffect(() => {
        if (inView) {
          if (
            !row.original.alerts
            || (Array.isArray(row.original.alerts) && row.original.alerts.length !== row.original.alert_counts?.all)
          ) {
            getIncidentAlerts(row.original.id);
          }
          if (!row.original.notes) {
            getIncidentNotes(row.original.id);
          }
        }
      }, [inView]);
```